src/assets/styles/main.scss

```diff
@@ -1,45 +1,58 @@
-$text-color: #0000;
+$default-black-color: rgba(0, 0, 0, 0.815);
 $primary-button-background: #d0295b;
 $button-text-color: #ffff;
+$input-focus__color: hsl(236, 87%, 51%);
+
 $home-card-container-shadow :0px 30px 20px -33px #a19f9f;
-$header-container-shadow :0px 30px 20px -36px #a19f9f;
+$header-container-shadow :0px 2px 10px 0px rgb(0 0 0 / 15%);
+$header-container-shadow-mobile :0px 2px 10px 0px rgb(0 0 0 / 15%);
+$bottom-border : 0.1rem;
+$bottom-border-solid : 0.1rem solid;
+$full-width:100%;
 
 $product-card-width-tablet: 15vh;
-$width-content-container:74rem;
-// $category-background-color: rgb(238, 237, 237);
+$width-content-container:64rem;
 $category-background-color: #ededed;
 $background-primary :  hsl(0, 0%, 100%);
 
 $font-vsmall: 0.7rem;
 $font-small: 0.8rem;
 $font-medium: 0.9rem;
 $font-large: 1rem;
+$font-vlarge: 1.1rem;
 
 // Viewport breakpoints
-$breakpoint-mobile: 480px;
-$breakpoint-tablet: 770px;
-$breakpoint-desktop: 1024px;
+$breakpoint-mobile: 481px;
+$breakpoint-tablet: 769px;
+$breakpoint-desktop: 1025px;
 
 $gen-pad-margin:0.5rem;
 $zero-margin-auto: 0 auto;
+$zero-margin: 0;
+$flex-grow-one: 1;
+$padding-one: 1rem;
+$max-height: 11rem; 
+$padding-main-logo: 0.4rem;
```

Comment:
variables & resets & common styles need segregation

Reply:
Will handle it going ahead.